cmd/notifier/config/config.toml

```diff
@@ -13,7 +13,14 @@
         Prefix = "erd"
         Length = 32
 
+    # CheckDuplicates signals if the events received from observers have been already pushed to clients
+    # Requires a redis instance/cluster and should be used when multiple observers push from the same shard
+    CheckDuplicates = true
+
 [WebSocketConnector]
+    # Enabled will determine if websocket connector will be enabled or not
+    Enabled = false
```

Comment:
do we have a check that a maximum of one of the connector types is enabled? I assume having both of them (or no one) active is a configuration issue

Reply:
no, i've added enabled flag for each section separately in order to be able to have this in place; it is useful for a smooth transition from http to ws integration

Reply:
due to deduplication we can handle multiple connectors at the same time

Reply:
after http integration will be fully removed, there will be again only one integration